platform/binance/model.go

```diff
@@ -64,6 +65,89 @@ type Tx struct {
 	Hash          string      `json:"txHash"`
 	Value         json.Number `json:"value"`
 	Memo          string      `json:"memo"`
+	HasChildren   int         `json:"hasChildren"`
+	SubTxsDto     SubTxsDto   `json:"subTxsDto"`
+}
+
+type SubTxsDto struct {
+	TotalNum     uint   `json:"totalNum"`
+	SubTxDtoList SubTxs `json:"subTxDtoList"`
+}
+
+type SubTx struct {
+	Hash     string      `json:"hash"`
+	Height   uint64      `json:"height"`
+	Type     TxType      `json:"type"`
+	Value    json.Number `json:"value"`
+	Asset    string      `json:"asset"`
+	FromAddr string      `json:"fromAddr"`
+	ToAddr   string      `json:"toAddr"`
+	Fee      json.Number `json:"fee"`
+}
+
+type SubTxs []SubTx
+
+func (subTxs *SubTxs) getTxs() (txs []Tx) {
+	mapTx := map[string]Tx{}
+	for _, subTx := range *subTxs {
+		key := subTx.ToAddr + subTx.Asset
+		tx, ok := mapTx[key]
+		if !ok {
+			mapTx[key] = subTx.toTx()
+			continue
+		}
+		txValue, err := tx.Value.Float64()
+		if err != nil {
```

Comment:
do we need this check? default txValue won't be 0?

Reply:
I can remove it, but I need to see the current implementation of `json.Number` to see if is safety